wp-content/themes/twentytwenty-child/functions.php

```diff
@@ -547,9 +546,13 @@ function show_dictionary_search() {
  * Function executed by AJAX to remotely get data from Oxford Dictionary
  * */
 function search_oxford_dictionary(){
+    $result = wp_remote_get( esc_url('https://www.oxfordlearnersdictionaries.com/definition/english/' . sanitize_title_with_dashes( $_POST['word']) ) );
 
-    $result = wp_remote_get( 'https://www.oxfordlearnersdictionaries.com/definition/english/' . sanitize_title_with_dashes( $_POST['word'] ) );
-    echo $result['body'];
+    if( is_wp_error( $result ) ) 
+        return;
+
+    $body = wp_remote_retrieve_body( $result );
+    echo $body;
 }
 add_action( 'wp_ajax_nopriv_search_oxford_dictionary', 'search_oxford_dictionary' );
```

Comment:
Isn't the search just inside the dashboard (correct me if I am wrong)? Why are we making the AJAX action public then?